release/mixer.sh

```diff
@@ -61,13 +64,22 @@ build_update() {
 }
 
 build_deltas() {
-    local mix_ver="$1"
```

Comment:
sweet, this was leftover from when we had to generate the +20 before the +10 for the format bump.

Reply:
I was on the fence about this too, and the mixer team said that it uses the current version that you're mixing by default, which made me think the --to was redundant.

Reply:
yep, lets remove it. We can always add it back if needed.